Makefile

```diff
@@ -155,7 +164,7 @@ bandit: requirements .bandit
 lint: requirements .lint
 
 .PHONY: .lint
-.lint: .flake8 .pylint .bandit .st2client-dependencies-check .st2common-circular-dependencies-check .rst-check
+.lint: .generate-openapi-spec .flake8 .pylint .bandit .st2client-dependencies-check .st2common-circular-dependencies-check .rst-check
```

Comment:
After thinking about it some more, I can't think of a better place.

We basically do a similar thing in st2docs - we generate files in `.docs` target. Those are files which always need to be re-generated and commited to the repo, same as in this case.